### ISSUE TITLE
Trouble compiling for Windows

### ISSUE THREAD

**Issue by vncastanheira** [klaussilveira/qengine#2]
So, after running CMAKE and creating the VS 2017 projects, the solution isn't compiling. It gives a ` invalid numeric argument /Wextra` for a lot of projects.
How to fix that?

**Comment by klaussilveira** [comment]
Binaries were only tested in MacOS and Linux. Some tweaks are certainly necessary for a Windows or mingw32 build.

**Comment by vncastanheira** [comment]
Yeah, I messed around with the parameters but C++ is not really my jam. It compiled at the end but the game crashed around second mission or so.
Gonna get back when I have the time to test it again.
